evaluate_model.py

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Please merge master into this branch `git merge origin master` and make sure everything still works